nautobot/extras/api/serializers.py

```diff
@@ -1,10 +1,12 @@
+from django.conf import settings
 from django.contrib.contenttypes.models import ContentType
 from django.core.exceptions import ObjectDoesNotExist
 from django.utils.functional import classproperty
 from drf_spectacular.utils import extend_schema_field
 from nautobot.core.api.serializers import BaseModelSerializer
 from nautobot.extras.models.secrets import SecretsGroupAssociation
 from rest_framework import serializers
+from rest_framework.reverse import reverse
```

Comment:
Interesting. I hadn't known about this function as an alternative to Django's one. I need to check whether `RelationshipsDataField.to_representation` can use this function as it would fix an issue I had with the URL serialization.